LLama/runtimes/build/LLamaSharp.Backend.Cpu.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>LLamaSharp.Backend.Cpu</id>
-    <version>$version$</version>
+    <version>0.11.2</version>
```

Comment:
Please revert this change, seems that it's modified when you were testing it.

Reply:
Oops, good catch!